docs/NewOperators.md

```diff
@@ -0,0 +1,18 @@
+## How to implement a new operator in Glow
+
+> Example PR: https://github.com/pytorch/glow/pull/2334
+
+### Implementation
+* Create a new glow high level IR node in `ClassGen/NodeGen.cpp`.
```

Comment:
This is not always the case. Sometimes it makes sense to do direct lowering, e.g., FCTransposed.

Reply:
super nit: Can we always capitalize "Glow" 😄